src/index.js

```diff
@@ -1,36 +1,54 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable import/prefer-default-export */
+
 import './style.css';
+import addToDo from './modules/addToDo.js';
+import displayToDo from './modules/displayToDo.js';
+import editToDo from './modules/editToDo.js';
+
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
 
-const toDoTask = [
-  {
-    description: 'Finish Carry Overs',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Go for a 15 min walk',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Finish this week project',
-    completed: false,
-    index: 2,
-  },
-];
+// Elements
 
+const form = document.getElementById('toDoForm');
 const table = document.getElementById('table-body');
 
-const displayList = () => {
-  for (let i = 0; i < toDoTask.length; i += 1) {
-    const row = document.createElement('tr');
-    row.innerHTML = `
-      <td>
-        <p><span>${toDoTask[i].completed} |</span> ${toDoTask[i].description}</p>
-      </td>
-      <td class="iAlign">${toDoTask[i].index}</td>
-    `;
-    table.appendChild(row);
-  }
+// Form Submit
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  addToDo();
+  displayToDo();
+  localStorage.setItem('todos', JSON.stringify(todos));
+});
+
+// Remove to do
+const removeToDo = (todoId) => {
+  todos = todos.filter((todo, index) => index !== todoId);
+  displayToDo();
+  localStorage.setItem('todos', JSON.stringify(todos));
 };
```

Comment:
- [ ] @HunterX-7 great work implementing your remove function, kindly move the function from your index.js file , the project requirements require it to be in a new file:
![image](https://user-images.githubusercontent.com/61408860/206840939-002925f1-dd5d-4cef-a1c7-589db2be50c9.png)